astropy/nddata/utils.py

```diff
@@ -509,17 +510,18 @@ def __init__(self, data, position, shape=None, side_length=None, wcs=None,
             `~astropy.coordinates.SkyCoord`, in which case ``wcs`` is a
             required input.
 
-        shape : tuple, optional
-            The shape (``(ny, nx)``) of the cutout array in pixel
-            coordinates (but see the ``mode`` keyword for additional
-            details). May be specified as a `~astropy.units.Quantity`
-            equivalent to pixels.
-
-        side_length : scalar, optional
-            The length (in pixel coordinates) of a side in a square cutout
-            array. ``shape`` will be set to ``(side_length, side_length)``.
-            See the ``mode`` keyword for additional details. May be specified
-            as a `~astropy.units.Quantity` equivalent to pixels.
+        size : int, array-like, `~astropy.units.Quantity`
+            The size of the cutout array along each axis.  If ``size``
+            is a scalar number or a scalar `~astropy.units.Quantity`,
+            then a square cutout of ``size`` will be created.  If
+            ``size`` has two elements, they should be in ``(ny, nx)``
```

Comment:
`size` was inherited from the original `shape` keyword (which is obviously in `(ny, nx)` order).  I considered switching the order, but the `size` keyword in `scipy.ndimage` functions (e.g. http://docs.scipy.org/doc/scipy-0.14.0/reference/generated/scipy.ndimage.filters.median_filter.html#scipy.ndimage.filters.median_filter) also use `(ny, nx)` (axis) order.  I thought it best to have a consistent API.